### PR TITLE
enableLogging flag

### PR DESCRIPTION
**Pull request by jamesallen-vol**
I added a flag to enable logging. I'm launching cloudformation templates via AWS Lambda functions so I don't need the logging since I'm picking up status from SNS notifications.

### CODE REVIEW

**Comment by andyday** [comment]
great addition.  i added something to the cleanup function called async.  by default it runs synchronously logging the results, but with async = true it runs async with no logging.  could do this the same way?  take a look at cleanup.  i'm open to names other than async...

**Comment by jamesallen-vol** [comment]
will do, thanks for pointing that out.

**Comment by jamesallen-vol** [comment]
I'm updating this per your suggestion. I think async works just as well and it keeps the code/interface more consistent.

**Comment by jamesallen-vol** [comment]
ok, I think that does it. If you see something that could be better let me know. Thanks for the quick response!

File: index.js [hunk 1]
```diff
@@ -234,7 +235,7 @@ function Cfn(name, template) {
         return Promise.resolve(JSON.stringify(fn(params)));
     }
 
-    function processStack(action, name, template) {
+    function processStack(action, name, template, async) {
```
Comment (andyday) [inline index.js]: since processStack is within the Cfn closure, async does not need to be passed as a param here, can just use async from the closure...

Comment (jamesallen-vol) [inline index.js]: Done! Thanks for that, I'm still fairly new to javascript.

**Comment by andyday** [comment]
published as cfn 0.14.0

**Pull request merged**